### PR TITLE
IntaRNA v2.2.0

### PR DESCRIPTION
**Pull request by martin-mann**
* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [ ] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [x] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

**Comment by martin-mann** [comment]
@bgruening seems the (gnu) m4 macros for boost-lib-detection are not finding the boost lib for linking (see according configure error 
```
checking whether the Boost::System library is available... yes
configure: error: Could not find a version of the library!
```
which I only see if the boost devel is not correctly installed.. :/  was working for boost-1.61 !?!

**Comment by martin-mann** [comment]
seems it might get solved by setting `configure --with-boost-libdir=` seed [ laanwj commented on 11 Nov 2013 ](https://github.com/bitcoin/bitcoin/issues/3219)

**Comment by martin-mann** [comment]
what is the boost lib install path in bioconda at the end?

File: recipes/intarna/build.sh [hunk 1]
```diff
@@ -33,7 +33,8 @@ fi
             --with-boost=$PREFIX \
             $CONFIGURE_MULTITHREADING \
             --disable-log-coloring \
-            ${extra_config_options} \
+            --with-boost-libdir=$PREFIX/lib64 \
```
Comment (bgruening) [inline recipes/intarna/build.sh]: $PREFIX/lib should be the path, you can look this up by simply untar the boost tarball.

**Comment by martin-mann** [comment]
@bgruening have seen in my prelast try.. ☺

**Comment by bgruening** [comment]
:+1:

**Pull request merged**